docs/en/operation/Journal.md

```diff
@@ -336,44 +160,6 @@ make the checkpointing a few times faster(depending how many threads are used).
 `alluxio.master.metastore.rocks.parallel.backup.compression.level` specifies the compression level, 
 where smaller means bigger file and less CPU consumption, and larger means smaller file and more CPU consumption. 
 
-#### Checkpointing on the leading master
-
-Checkpointing requires a pause in master metadata changes and causes temporary service
-unavailability while the leading master is writing a checkpoint. 
-This operation may take hours depending on Alluxio's namespace size.
-Therefore, Alluxio's leading master will not create checkpoints by default.
-
-Restarting the current leading master to transfer the leadership to another running master periodically
-can help avoid leading master journal logs from growing unbounded when Alluxio is running in HA mode.
-
-Starting from version 2.4, Alluxio embedded journal HA mode supports automatically transferring checkpoints from standby masters to the leading master. 
-The leading master can use those checkpoints as taken locally to truncate its journal size without causing temporary service unavailability. 
-No need to manually transfer leadership anymore.
-
-If HA mode is not an option, the following command can be used to manually trigger the checkpoint:
-
-```shell
-$ ./bin/alluxio fsadmin journal checkpoint
```

Comment:
I notice that we are not doing checkpointing in leading master... Which means the journal would be growing slowly if there's only one master and eventually cause problem. Is it true? @yuzhu @Xenorith @beinan

Reply:
i don't have an answer to this question, but i also don't think it's blocking this PR. if determined that the command is needed, we'd need to add it back outside of this PR